README.md

```diff
@@ -0,0 +1,37 @@
+# Notebook data redirector
+
+This is an AWS application that redirects requests to files publicly hosted on Box.
+
+## Deployment
+
+To deploy the redirector, first install and configure the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html).
+
+Next, run the following commands:
+
+```console
+$ sam build
+$ sam package --s3-bucket your-s3-bucket --output-template packaged.yaml
```

Comment:
is your S3 bucket an existing bucket, or will it be created?

Reply:
I used the dmd-test-cf-packages s3-bucket but I'm getting an error in the sam package step. "Unable to upload artifact BoxWebhookFunction referenced by CodeUri parameter of BoxWebhookFunction resource.
An error occurred (AccessDenied) when calling the PutObject operation: Access Denied"

Reply:
my bad, using the wrong aws cli profile